python/GangaAtlas/Lib/Athena/DQ2JobSplitter.py

```diff
@@ -599,6 +601,7 @@ def split(self,job):
             allowed_sites.append('PRAGUELCG2-RUCIOTEST_DATADISK')
 
         logger.debug('allowed_sites = %s ', allowed_sites)
+        logger.warning('allowed_sites = %s ', allowed_sites)
```

Comment:
A general comment on the additional logger messages: do you want these kept in? It doesn't really matter as HC are the only ones using Panda/DQ2JobSplitter but I thought it might make things a bit noisy!